qhub/schema.py

```diff
@@ -186,6 +188,15 @@ class Main(Base):
     ci_cd: CiEnum
     security: Security
     profiles: Profiles = []
+    terraform_version: str
+
+    @pydantic.validator("terraform_version")
+    def verify_terraform_version(cls, v):
```

Comment:
I think we should have terraform version as optional, otherwise this would not be backwards compatible.